crates/jstz_cli/src/repl/js_logger.rs

```diff
@@ -0,0 +1,24 @@
+use boa_engine::Context;
+use jstz_api::js_log::{JsLog, LogData};
+use jstz_core::runtime::with_global_host;
+use tezos_smart_rollup::prelude::debug_msg;
+
+pub struct PrettyLogger;
+
```

Comment:
Please mention addition of PrettyLogger and its functionality in the PR description.

Reply:
`PrettyLogger` is already part of [`main`](https://github.com/trilitech/jstz/blob/main/crates/jstz_cli/src/js_logger.rs)